### DISCUSSION TITLE
Olop Early Acces pour Windows

### DISCUSSION THREAD

**Discussion by SuperAtraction** [OlopTech/Olop#1] [Announcements]
Bonjour à toutes et à tous,
Je suis ravi de vous annoncer qu'une préversion d'Olop pour Windows est désormais disponible ! Cette version inclut :

- Une partie du système InstallPass, qui vous permet d'installer des applications en utilisant un simple code
- Notre système de fenêtrage, y compris une fonction de minimisation
- La nouvelle interface d'Olop BB (Bleu Blanc)

⚠️ Avertissement : Veuillez noter que cette version est un accès anticipé, ce qui signifie qu'elle pourrait être instable. De plus, elle utilise Qt 6.5.1 au lieu de Qt 6.5.2, ce qui signifie qu'elle n'inclut pas les corrections de 291 bugs et des problèmes de sécurité qui ont été résolus dans la version plus récente. Elle utilise également une version plus ancienne de QWebEngine, le moteur web de Qt. Elle vous avertira qu'une mise à jour est disponible, mais c'est faux. Normalement, le système de mises à jour sera opérationnel pour l'alpha 1.1. 

Le lien : [http://olop.rf.gd/index.html?url=download.html](https://olop.rf.gd/index.html?url=download.html)

**Comment by jeanne123456789** [comment]
mais c'est génial tout ça ! Bon j'attends ton alpha 1.1. parce que je n'ai pas envie de "planter" mon PC

**Reply by SuperAtraction** [reply]
Non, c'est bon… C'est juste que mon logiciel n'est pas complet !

**Comment by SuperAtraction** [comment]
Actuellement, il y a un bug avec proton drive et le lien. (Notez que la prochaine version windows sera ***normalement*** pour l'alpha 1.2)
